src/pages/library/RequestDatasetAccessModal.js

```diff
@@ -103,6 +103,8 @@ const RequestDatasetAccessButton = ({ title, id, setShowWipModal }) => {
   return h(ButtonPrimary, {
     disabled: status,
     onClick: withErrorReporting('Error requesting dataset access', async () => {
+      // Should this point to catalog? My understanding is that we are fully public?
```

Comment:
This is the only thing I am really not sure on the behavior of. The modal is only used in the Data Browser, so from what I can tell it should probably point to catalog, but I wasn't sure because I feel like someone said all catalog datasets were public for now.

Reply:
yes, this should point towards the catalog! Right now, it shoudl also be pointing *nowhere* haha.